charts/sdfactory/README.md

```diff
@@ -19,23 +19,26 @@ Helm Charts for SD Factory application. Self-Description Factory component is re
 | autoscaling.targetCPUUtilizationPercentage | int | `80` |  |
 | fullnameOverride | string | `""` |  |
 | image.pullPolicy | string | `"Always"` | Set the Image Pull Policy |
-| image.repository | string | `"ghcr.io/catenax-ng/tx-sd-factory/sdfactory"` | Image to use for deploying an application |
-| image.tag | string | `""` | Image tage is defined in chart appVersion. |
+| image.repository | string | `"tractusx/sdfactory"` | Image to use for deploying an application |
+| image.tag | string | `"main"` | Image tage is defined in chart appVersion. |
```

Comment:
Do not use main as the default tag for the image. Leave it blank and set it in the template to the Chart.yaml appVersion